src/midi/midi_fluidsynth.cpp

```diff
@@ -22,25 +22,25 @@
 
 #if C_FLUIDSYNTH
 
-#include <bitset>
-#include <cassert>
-#include <deque>
-#include <numeric>
-#include <string>
-#include <tuple>
-
-#include "../ints/int10.h"
-#include "ansi_code_markup.h"
-#include "channel_names.h"
-#include "control.h"
-#include "cross.h"
-#include "fs_utils.h"
-#include "math_utils.h"
-#include "mixer.h"
-#include "pic.h"
-#include "programs.h"
-#include "string_utils.h"
-#include "support.h"
+	#include <bitset>
+	#include <cassert>
+	#include <deque>
+	#include <numeric>
+	#include <string>
+	#include <tuple>
```

Comment:
I would remove these indents.  `#include`'s are never indented in the rest of our codebase and I don't think we want to be indenting the entire file because of a top level `#if C_FLUIDSYNTH` that applies to this entire file.

Reply:
Yeah, this is clang-reformat, and it annoys me a bit too. I enabled indenting nested preprocessor directives a while ago:
https://clang.llvm.org/docs/ClangFormatStyleOptions.html#indentppdirectives

That works well for stuff like this:

```
#if FOO
  #if BAR
    #include <foo>
  #endif
#endif
```

But not in other cases, apparently. It's not smart enough to DoTheRightThing(tm), so I might disable this feature as I rely on mass-formatting files.

Reply:
Disabled the indenting of preprocessor directives & reformatted.